components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/org/wso2/carbon/idp/mgt/ui/i18n/Resources.properties

```diff
@@ -175,6 +175,8 @@ passive.sts.config=WS-Federation (Passive) Configuration
 passive.sts.local.config=WS-Federation (Passive) Configuration
 sts.local.config=Security Token Service Configuration
 passive.sts.enabled=Enable Passive STS
+oidc.scopes=Scopes
+oidc.scopes.help=Enter OIDC Scopes
```

Comment:
Here also we can simply keep as scope. Any reason we need to mention as oidc scopes?

Reply:
I will update it to just `scopes`